pages/_conferenceCode/contacts/index.vue

```diff
@@ -366,18 +392,100 @@ function isFiltersApplied (_this){
   else return false
 }
 
+async function asyncFind (query){
+  this.isLoadingOrganization = true
+  if (query !== '')
+    this.organizationOptions = await this.$kronosApi.getOrganizations({
+      FreeText: query,
+      limit   : 25
+    })
+
+  this.isLoadingOrganization = false
+}
+
 async function mounted (){
+  const qs = this.$route.query
+
   this.meetingsOptions = await getMajorMinorMeetings(this)
 
-  this.organizationOptions = await this.$kronosApi.getOrganizations({
-    limit: 5000
-  })
+  // this.organizationOptions = await this.$kronosApi.getOrganizations({
+  //   limit: 25
+  // })
+
+  this.countryList = this.$store.getters['countries/asSelectOptions']
+
+  sortCountryByAsc(this.countryList, this)
+
+  this.organizationTypesOptions = this.$store.getters[
+    'organizations/asSelectOptions'
+  ]
+
+  setQueryStringFilters(this, qs)
+}
+
+function setQueryStringFilters (_this, qs){
+  if (qs.FreeText) _this.searchTagInput = qs.FreeText
+
```

Comment:
do not pass `this` as a parameter, better to bind the function to `this`.

```suggestion
  setQueryStringFilters.apply(this, qs)
}

function setQueryStringFilters (qs){
  if (qs.FreeText) this.searchTagInput = qs.FreeText
...
```

Reply:
done

Reply:
yes it's `call`. with `apply` you need to pass argument in array. I always mixing them